README.md

```diff
@@ -11,18 +11,31 @@ to perform various operations on them.
 
 ## Perform Operations on Arrays
 
-We're going to write code to get all of the following results from the integers
-in our array:
+Your task is to write a series of methods that extract specific values from an
+input array of integers. These methods should work with any array of integers,
+but one is provided as an example to work with on the first line of
+`lib/array_practice.rb`:
 
-- all of the odd numbers all of the even numbers the square of all the numbers
-- the first number whose square is > 350 all the numbers whose square is > 350
-- all the numbers, cubed the first number whose cube is > 500 all the numbers
-- whose cube is < 500
+```ruby
+array_of_integers = *0..50
+# => [0, 1, 2, 3, 4, 5, 6, 7, 8, 9,  ...  47, 48, 49]
+```
+
+> This method of array creation uses a 'splat' operator on a [range]. This is a
+> shorthand method for creating and assigning an array of integers
+
+Below this are the methods you must write implementation for. Read the provided
```

Comment:
Is there something that should be below?

Reply:
Aye, took this out